src/vos/tests/vts_io.c

```diff
@@ -2549,32 +2549,32 @@ run_io_test(daos_ofeat_t feats, int keys, bool nest_iterators, const char *cfg)
 
 	init_num_keys = VTS_IO_KEYS;
 
-	feats = feats & DAOS_OF_MASK;
-	if ((feats & DAOS_OF_DKEY_UINT64) && (feats & DAOS_OF_DKEY_LEXICAL)) {
+	type = type & DAOS_OF_MASK;
+	if ((type & DAOS_OF_DKEY_UINT64) && (type & DAOS_OF_DKEY_LEXICAL)) {
```

Comment:
Don't you need to better use the "..._OT..." versions of defines ?

Reply:
Thank you! Yes. I will use *_OT_* and remove _OT_ version definition. I also need to add "DAOS_OT_MASK" to replace "DAOS_OF_MASK".